charts/hail-search/templates/persistentvolume.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.persistentVolume.enabled -}}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ include "hail-search.fullname" . }}-pv
+  labels:
+    {{- include "hail-search.labels" . | nindent 4 }}
+spec:
+  storageClassName: ""
+  capacity:
+    storage: {{ .Values.storageCapacity }}
+  accessModes:
+    - ReadOnlyMany
+  volumeMode: Filesystem
+  persistentVolumeReclaimPolicy: Retain
+  claimRef:
+    name: {{ include "hail-search.fullname" . }}-pvc
+  csi:
+    driver: {{ .Values.persistentVolume.driver }}
+    volumeHandle: {{ .Values.persistentVolume.volumeHandle }}
+    fsType: 'ext4'
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "hail-search.fullname" . }}-pvc
+  labels:
+    {{- include "hail-search.labels" . | nindent 4 }}
+spec:
+  storageClassName: ""
+  volumeMode: Filesystem
+  volumeName: {{ include "hail-search.fullname" . }}-pv
+  accessModes:
+    - ReadOnlyMany
+  resources:
+    requests:
+      storage: 2000G
```

Comment:
unless we don't expect this to ever change (or if it simply needs to match the capacity attribute on the persistentvolume), i'd make this a variable as well. I don't have a strong preference here, but this unit could also be 2T (or 2Ti if you need the 1024-ish variant of a terabyte).

Reply:
🤦 this is actually a bug.  I'm using `{{ .Values.storageCapacity }}` elsewhere in this file and just missed this one.

Reply:
the values don't _need_ to be the same, but they might as well be for simplicity.